app/views/activities/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="l-header-single -activities-article" <% if @activity.picture_file_name %>style="background-image: url('<%= @activity.picture %>');"<% end %>>
+<div class="l-header-single -activities-article" <% if @media_content.mediable_photo_url.present? %>style="background-image: url('<%= @media_content.mediable_photo_url %>');"<% end %>>
```

Comment:
Might be worth checking if @media_content exist:

`if @media_content && @media_content.mediable_photo_url.present?`